tests/test_mig_shared_functionality_cat.py

```diff
@@ -32,12 +32,13 @@
 import os
 import shutil
 import sys
+import unittest
 
-from tests.support import MIG_BASE, TEST_DATA_DIR, MigTestCase, testmain, \
+from tests.support import MIG_BASE, PY2, TEST_DATA_DIR, MigTestCase, testmain, \
     fixturefile, fixturefile_normname, ensure_dirs_exist, temppath
 
 from mig.shared.base import client_id_dir
-from mig.shared.functionality.cat import _main as main
+from mig.shared.functionality.cat import _main as main, main as cgimain
```

Comment:
Nit-pick but I think this is a bad choice of naming. Importing and using `_main` as `main` when there's actually a real `main` there is confusing and `cgimain` is misleading as it is shared for all handlers. Only the actual call arguments differ when used from CGI (e.g with environ as noted elsewhere).
Sometimes minimizing diff is good for PR reviews, but I'll probably rename during merge to reduce confusion and prevent the pattern from spreading next time we need some functionality unit test.

Reply:
Renamed during merge.